app/src/main/java/net/kuama/documentscanner/presentation/BaseScannerActivity.kt

```diff
@@ -12,13 +13,17 @@ import androidx.lifecycle.Observer
 import kotlinx.android.synthetic.main.activity_scanner.*
 import net.kuama.documentscanner.R
 import net.kuama.documentscanner.data.Loader
+import net.kuama.documentscanner.domain.Failure
+import net.kuama.documentscanner.domain.PerspectiveTransform
 import java.io.File
 
+@androidx.camera.core.ExperimentalGetImage
 abstract class BaseScannerActivity : AppCompatActivity() {
     private lateinit var viewModel: ScannerViewModel
+    private val perspectiveTransform: PerspectiveTransform = PerspectiveTransform()
+    private var thresholdValue = 0
```

Comment:
If we don't use `thresholdValue` we should get rid of this field and of the slider inside the UI